src/Queue/Task.php

```diff
@@ -95,9 +95,10 @@ public function __construct(?Io $io = null, ?LoggerInterface $logger = null) {
 		$this->io = $io ?: new Io(new ConsoleIo());
 		$this->logger = $logger;
 
-		$this->loadModel($this->queueModelClass);
+		$tableLocator = $this->getTableLocator();
+		$this->QueuedJobs = $tableLocator->get($this->queueModelClass);
```

Comment:
Not really sure how we should tackle this. Should we remove that property, set it hardcoded as string and therefore remove the posibility to set a custom QueueModelClass?

Reply:
Could you please try the following? Not really sure if it would work.
`
        /**
         * @var \Queue\Model\Table\QueuedJobsTable
         */
        $QueuedJobs = $tableLocator->get($this->queueModelClass);

        $this->QueuedJobs = $QueuedJobs;
`

Reply:
Looks pretty good! Thanks 👍🏻